source/mainnet/smart-contracts/tutorials/eSealing/eSealing_dapp.rst

```diff
@@ -0,0 +1,89 @@
+.. include:: ../../../variables.rst
+.. _eSealing_dapp:
+
+=============
+eSealing dApp
+=============
+
+You can explore the `hosted eSealing service <https://esealing.testnet.concordium.com>`_ or
+start your own front-end by following the instructions in the READ.me file of this
+`repository <https://github.com/Concordium/concordium-browser-wallet/tree/main/examples/eSealing>`_ to get an overview of the dApp flow.
+
+.. note::
+
+   Comprehensive instructions on how to set up the |bw|, create an account in the |bw|,
+   get some testnet CCD and run a local front-end can be found in :ref:`wCCD front-end-set-up section <wCCD-front-end-set-up>`
+
+The front end supports the following two flows with the |bw| (or |mw-gen2| that uses WalletConnect):
+
+- Compute the file hash of a selected file => register its file hash in the smart contract
+- Compute the file hash of a selected file => retrieve the timestamp and witness (sealer account) that registered the file hash
+
+.. note::
+
+   Select the ``registration tab`` to register the file hash and the ``display tab`` to look up the
```

Comment:
We use bold text in a few places to refer to things that users should click on. Perhaps we should also do that here. Do we have any guidelines for that @dg-concordium ?

Reply:
@dg-concordium Any guidelines/preferences please?